Libplanet.Action/State/ValidatorSetAccount.cs

```diff
@@ -0,0 +1,60 @@
+using Libplanet.Crypto;
+using Libplanet.Store.Trie;
+using Libplanet.Types.Blocks;
+using Libplanet.Types.Consensus;
+
+namespace Libplanet.Action.State
+{
+    public class ValidatorSetAccount
```

Comment:
Could you add a more detailed comment?

Reply:
Eh, sure. I just added a short description. Technically, this isn't supposed to be `public` considering how it is implemented. 🙄